search/search_indexes.py

```diff
@@ -33,14 +43,46 @@ def index_queryset(self, using=None):
 
     def prepare_text(self, obj):  # pylint: disable=no-self-use
         """Indexing of the primary content of a LearningResource."""
+        try:
+            # Strip XML tags from content before indexing.
+            tree = etree.fromstring(obj.content_xml)
+            content = etree.tostring(tree, encoding="utf-8", method="text")
+        except etree.XMLSyntaxError:
+            # For blank/invalid XML.
+            content = obj.content_xml
```

Comment:
Should we log somewhere that this is happening?
Maybe could even show the user that this is happening (in another issue, since here it's out of scope)